packages/tx/src/eip4844Transaction.ts

```diff
@@ -0,0 +1,479 @@
+import { byteArrayEquals, toHexString } from '@chainsafe/ssz'
```

Comment:
Out of interest: we do not have equivant util functions in our own Util package?